test/validation_test.py

```diff
@@ -147,3 +147,7 @@ def test_return_false_if_the_date_string_contain_the_date_format_year_month_day_
         value_example = "2018/16/03"
         value_boolean = validation.is_valid_date(value_example)
         self.assertFalse(value_boolean)
+
+
+if __name__ == "__main__":
+    unittest.main()
```

Comment:
@ovandononi @Miquilin @jpsandovaln 
I think I'm going to start a debate about whether we should or not use this block 
`if __name__ == "__main__":
    unittest.main()`

Reply:
I was checking documentation about it and seems that it is used to run each test file from commad line, but I am not sure if it should be there. @jpsandovaln could you please help us with this doubt?

Reply:
Those lines were a suggestion of Raul. Those allow see if there is test cases failed or do not.